README.md

```diff
@@ -101,26 +107,111 @@ export function loadConfig(oidcConfigService: OidcConfigService) {
 export class AppModule {
     constructor(private oidcSecurityService: OidcSecurityService, private oidcConfigService: OidcConfigService) {
         this.oidcConfigService.onConfigurationLoaded.subscribe(() => {
-            const config = new OpenIDImplicitFlowConfiguration();
-            config.stsServer = 'https://your_secure_token_service_url';
-            config.redirect_url = 'https://localhost:4200';
-            config.client_id = 'angular_code_client';
-            config.scope = 'openid profile email';
-            config.response_type = 'code';
+          const config = new OpenIDImplicitFlowConfiguration();
+          //merge configuration loaded from assets/auth.clientConfiguration.json
+          Object.assign(implicitFlowConfig, this.oidcConfigService.clientConfiguration);
+          this.oidcSecurityService.setupModule(config, this.oidcConfigService.wellKnownEndpoints);
+        });
+    }
+}
+```
 
-            config.silent_renew = true;
-            config.silent_renew_url = 'https://localhost:4200/silent-renew.html';
+#### assets/auth.clientConfiguration.json
 
-            const authWellKnownEndpoints = new AuthWellKnownEndpoints();
-            authWellKnownEndpoints.setWellKnownEndpoints(this.oidcConfigService.wellKnownEndpoints);
+See [Auth documentation](https://github.com/damienbod/angular-auth-oidc-client/blob/master/API_DOCUMENTATION.md#authconfiguration)
+for the detail of each field.
 
-            this.oidcSecurityService.setupModule(config, authWellKnownEndpoints);
-        });
+```json
+{
+	"stsServer": "https://localhost:44318",
+	"redirect_url": "https://localhost:44311",
+	"client_id": "angularclient",
+	"response_type": "code",
+	"scope": "dataEventRecords securedFiles openid profile",
+	"post_logout_redirect_uri": "https://localhost:44311",
+	"start_checksession": true,
+	"silent_renew": true,
+	"silent_renew_url": "https://localhost:44311/silent-renew.html",
+	"post_login_route": "/home",
+	"forbidden_route": "/forbidden",
+	"unauthorized_route": "/unauthorized",
+	"log_console_warning_active": true,
+	"log_console_debug_active": true,
+  "max_id_token_iat_offset_allowed_in_seconds": 10,
+}
+```
+At present only the 'code' with PKCE, 'id_token token' or the 'id_token' flows are supported:
+
+`"response_type": ["code" | "id_token token" | "id_token" | "token" ]`
```

Comment:
can you remove the "token", this is not supported

Reply:
`"response_type": ["code" | "id_token token" | "id_token"  ]`

Reply:
done.